hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMAllocateBlockRequest.java

```diff
@@ -174,6 +175,12 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
     Result result = null;
 
     try {
+      ResolvedBucket bucket = ozoneManager.resolveBucketLink(keyArgs);
+      keyArgs = bucket.update(keyArgs);
```

Comment:
Looks like this logic is needed for all KeyRequests. Can we move this to a common method, instead of duplicating it.

Reply:
Thanks, extracted to a method.